web/themes/custom/novel/templates/components/list-teaser.html.twig

```diff
@@ -0,0 +1,49 @@
+{%
+  set classes = [
+  'event-list-item',
+  'arrow__hover--right-small',
+]
+%}
+
+<a {{ attributes.addClass(classes).setAttribute('href', url) }}>
+  <div class="event-list-item__image-container">
+    {{ image }}
+  </div>
+  <div class="event-list-item__content">
+    {{ categories }}
+    {% if date %}
+      <div class="event-list-item__date">
+        {{ date }}
+      </div>
+    {% endif %}
+
+    <h3 class="event-list-item__title">{{ title }}</h3>
```

Comment:
If possible could you change this to a `<h2>`?

It is out of scope for this PR but here we are looking at the DOM structure for these lists. Currently we want the list title ("Arrangementer" or "Biblioteker") to be the H3. This makes the heading level for each list item a `<h2>`. This would allow us to address #670 😅 .